CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/QualityControlViewPane.java

```diff
@@ -360,6 +367,10 @@ public void updateItem(final QualityControlEvent item, final boolean empty) {
         });
     }
 
+    public void setFirstClick(final boolean click) {
+        firstClick = click;
+    }
```

Comment:
Do not need a publicly accessible setter for this variable.
Assign the values directly.